modules/pages/client/components/FaqFoundation.component.js

```diff
@@ -123,6 +114,13 @@ export default function FaqFoundation() {
         .
       </div>
 
+      <div className="faq-question" id="who-are-the-board">
+        <h3>{t('Who are the board?')}</h3>
+        <Trans t={t} ns="pages">
+          <a href="/foundation#board">We</a>: Mikael and Natalia.
+        </Trans>
+      </div>
+
```

Comment:
I would replace the Q&A _Who are the board?_ with _Who operates Trustroots servers?_ or something similar.

As for me, there are two important pieces of information here: (1) who maintains and can troubleshoot the servers in case something happens; (2) who has access to the data, relevant from data privacy/protection perspective.

Former Board Members are listed and presented on the https://www.trustroots.org/foundation page (which needs updating too, BTW). 

There could be a Q&A _Who founded Trustroots?_ with an answer giving credits to all founders, possibly with the years when they were board members.

Reply:
I just noticed that there's the link https://team.trustroots.org/Server.html where the folks who operate Trustroots server are listed. That's great to have it there.